### ISSUE TITLE
Cannot detect which column is clicked inside grid's row

### ISSUE THREAD

**Issue by lanmaster** [vaadin/vaadin-grid-flow#671]
Flow 13.0.8. It should be very useful if we can get information about clicked column (bean property).
For example: 
```
component.addColumn(SomeBean::someProperty)
.addColumnClickListener((ClickedItem<SomeBean>) event -> {
   boolean isDoubleClicked = event.isDoubleClicked();
});
```

Or how i can make my own component with double-click listener ability?

**Comment by Springrbua** [comment]
This will probably be solved with #608

**Comment by JulianFeinauer** [comment]
Hey @Springrbua , is this solved now as #608 got merged. I am also interested in that : )

**Comment by pekam** [comment]
We will add API for retrieving the `Column`-reference from the existing events `ItemClickEvent` and `ItemDoubleClickEvent`. So I'm closing this issue in favor of #725.
